### ISSUE TITLE
Add IP v6 iptable rules

### ISSUE THREAD

**Issue by dolanjs** [freedomofpress/securedrop#1046]
reported by @iSec #iSEC-15FTC-3

IPv6 is disabled thru systctl settings but the default ip6table is set to allow for each of the IPv6 default chains. These should be changed from accept to drop for defense in depth in the event that IPv6 doesn't get disabled.

```
root@mon-staging:/home/vagrant# ip6tables -L
Chain INPUT (policy ACCEPT)
target prot opt source destination

Chain FORWARD (policy ACCEPT)
target prot opt source destination

Chain OUTPUT (policy ACCEPT)
target prot opt source destination
```

**Comment by dolanjs** [comment]
Per @iSec's audit should also change the default actions for the iptables chains in the IPv4 rules.

**Comment by dolanjs** [comment]
This was fixed in #1053 and merged into the release/0.3.4 branch.
